requirements.txt

```diff
@@ -1,8 +1,8 @@
-Flask>=2.2
+Flask>=2.2,<3
 
 dimod>=0.10.0
 dwave-system>=1.3.0
-dwave-cloud-client>=0.8.3
+dwave-cloud-client>=0.11.0,<0.12.0
```

Comment:
Hopefully we find a better namespace for this server to live in by 0.12. :slightly_smiling_face: